### PR TITLE
Addressing the review comments

### PR DESCRIPTION
**Pull request by AnjanaSamindraPerera**

### CODE REVIEW

File: components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java [hunk 1]
```diff
@@ -65,7 +65,8 @@ public void handleEvent(Event event) throws IdentityEventException {
                         getOrganizationManager().getOrganizationDepthInHierarchy(organization.getId());
             } catch (OrganizationManagementServerException e) {
                 throw new IdentityEventException(
-                        "An error occurred while getting depth of the sub-organization", e);
+                        String.format("An error occurred while getting depth of the sub-organization with id: %s",
```
Comment (sadilchamishka) [inline components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java:68]:
```suggestion
                        String.format("An error occurred while getting depth of the organization with id: %s",
```

File: components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java [hunk 2]
```diff
@@ -88,7 +89,8 @@ private void updateGovernanceConnectorProperty(Organization organization)
             identityGovernanceService.updateConfiguration(tenantDomain, configurationDetails);
         } catch (IdentityGovernanceException | OrganizationManagementException e) {
             throw new IdentityEventException(
-                    "An error occurred while enabling password recovery for sub-organization", e);
+                    String.format("An error occurred while enabling password recovery for sub-organization with id: %s",
```
Comment (sadilchamishka) [inline components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java:92]:
```suggestion
                    String.format("An error occurred while enabling password recovery for organization with id: %s",
```

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java:92]: These comments added as we try to move away from the sub-org keyword to organizations

File: components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java (hunk 1 above)
Comment (AnjanaSamindraPerera) [reply components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java]: Resolved in [4bc6161](https://github.com/wso2-extensions/identity-organization-management/pull/294#:~:text=to%20new%20term-,4bc6161,-Merge%20state)

File: components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java (hunk 2 above)
Comment (AnjanaSamindraPerera) [reply components/org.wso2.carbon.identity.organization.management.handler/src/main/java/org/wso2/carbon/identity/organization/management/handler/GovernanceConfigUpdateHandler.java]: Resolved in [4bc6161](https://github.com/wso2-extensions/identity-organization-management/pull/294#:~:text=to%20new%20term-,4bc6161,-Merge%20state)

**Review by sadilchamishka — APPROVED** [review]

**Pull request merged**